helm/values.yaml

```diff
@@ -0,0 +1,32 @@
+# Default values for medplum.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+namespace: medplum
+
+global:
+  cloudProvider: gcp
+  gcp:
+    projectId: medplum-zencore
+    secretId: config-diego-test
+ 
+deployment:
+  replicaCount: 1
+  image:
+    # repository: medplum/medplum-server
+    repository: us-west1-docker.pkg.dev/medplum-zencore/medplum-repo/medplum-server
+    tag: test4
```

Comment:
leave the default image with the app version tag

Reply:
latest?